Sources/Commands/Workspace.swift

```diff
@@ -408,6 +414,68 @@ public class Workspace {
         try saveState()
     }
 
+    /// Pins the repository at a given version.
+    ///
+    /// - Parameters:
+    ///   - dependency: The dependency to pin.
+    ///   - packageName: The name of the package which is being pinned.
+    ///   - at: The version to pin at.
```

Comment:
Nit: I believe the names in comments are the parameter names, not the external names.